### PR TITLE
Update .gitignore

### PR DESCRIPTION
**Pull request by MaTriXy**
added some files that were created while compiling redex to gitignore

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -30,3 +30,10 @@
 /redexdump
 /stamp-h1
 /test-driver
+
+
+redex-src-strings-map.txt
+pyredex/__pycache__/__init__.cpython-36.pyc
```
Comment (justinjhendrick) [inline .gitignore]: nit-pick: why not ignore all `.pyc` files?

File: .gitignore [hunk 2]
```diff
@@ -30,3 +30,10 @@
 /redexdump
 /stamp-h1
 /test-driver
+
+
```
Comment (justinjhendrick) [inline .gitignore]: nit-pick: one separating line is enough

File: .gitignore (hunk 1 above)
Comment (MaTriXy) [reply .gitignore]: I wasn't sure if any of them would actually be needed in the future.

File: .gitignore (hunk 2 above)
Comment (MaTriXy) [reply .gitignore]: kk.

**Pull request closed without merging**